addons/stock/stock_view.xml

```diff
@@ -1024,6 +1095,13 @@
             }
             </field>
             <field name="search_view_id" ref="view_picking_internal_search"/>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click here to create a new transfer.  
+              </p><p>
+                You can either do it immediatly or mark it as ToDo and then process it later. Use your scanner to validate the transferred quantity quicker.
```

Comment:
..." or mark it as Todo for future processing", maybe ? To propagate to other strings if modified.